metcalcpy/util/ecnt_statistics.py

```diff
@@ -263,17 +291,10 @@ def calculate_ecnt_rmse(input_data, columns_names, aggregation=False):
 
         Returns:
             calculated ECNT_RMSE as float
-            or None if some of the data values are missing or invalid
+            or None if some data values are missing or invalid
     """
-    warnings.filterwarnings('error')
-    try:
-        total = get_total_values(input_data, columns_names, aggregation)
-        rmse = math.sqrt(sum_column_data_by_name(input_data, columns_names, 'mse') / total)
-        result = round_half_up(rmse, PRECISION)
-    except (TypeError, ZeroDivisionError, Warning, ValueError):
-        result = None
-    warnings.filterwarnings('ignore')
-    return result
+
+    return weighted_average(input_data, columns_names, 'mse', aggregation)
```

Comment:
@TatianaBurek good idea writing/using a utility function to handle the common computation of weighted averages. I'm a bit suspicious on this one though. I think we need to compute the weighted average of the MSE and the compute the square root of the result and return that. That's what the old code was doing... so you just need to port back over that previous logic.